src/components/calculator.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+class Calcul extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calculator">
+        <input type="number" className="input" placeholder="0" />
+        <ul className="ac">
+          <button type="button">ac</button>
+          <button type="button">+/-</button>
+          <button type="button">%</button>
+          <button className="bg-orange" type="button">+</button>
```

Comment:
Please ensure that the Calculator.js component should match the following given design.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/159707914-b4c73363-5b61-48fc-bdae-34c8fa837ba1.png) | ![image](https://user-images.githubusercontent.com/10332499/159707998-315621d7-1299-4a34-b424-34c42971aebf.png)